user_docs/en/userGuide.t2t

```diff
@@ -1576,6 +1576,16 @@ It has the following options:
 - Beep and speak: This option tells NVDA to both beep and speak when a progress bar updates.
 -
 
+==== Progress bar speech percentage Interval ====[ObjectPresentationReportProgressBarSpeechPercentageInterval]
+This setting allows you to set after which percentage change the new progress bar state will be read by the synthesizer.
+
+==== Progress bar Beep percentage Interval ====[ObjectPresentationReportProgressBarBeepPercentageInterval]
+This setting allows you to set after which percentage change the new progress bar state will be indicated by the beep sound.
+
+==== Progress bar minimum beep frequency (HZ) ====[ObjectPresentationReportProgressBarMinimumBeepFrequency (HZ)]
+Here you can set what will be the initial frequency (in Hz), which will be used for the audio presentation of the progress bars.
+The default value of 110Hz corresponds to the sound A in the great octave.

```

Comment:
This also sounds familiar to people who have affinity with music, but what if one changes the minimum frequency to an e? how would they know what frequency to input?